storm-multilang/dotnet/src/main/dotnet/Dotnet.Storm.Adapter/Components/StormTuple.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Dotnet.Storm.Adapter.Messaging;
+using System.Collections.Generic;
+
+namespace Dotnet.Storm.Adapter.Components
+{
+    public class StormTuple
+    {
+        internal StormTuple(ExecuteTuple tuple)
+        {
+            Id = tuple.Id;
+            Component = tuple.Component;
+            TaskId = tuple.TaskId;
+            Stream = tuple.Stream;
+            Tuple = tuple.Tuple;
+        }
+
+        public string Id { get; internal set; }
+
+        public string Component { get; internal set; }
+
+        public string TaskId { get; internal set; }
+
+        public string Stream { get; internal set; }
+
+        public List<object> Tuple { get; internal set; }
```

Comment:
As it returns all values packed in a StormTuple, maybe GetValues() sounds like a more friendly name

Reply:
Tuple is Storm terminology. I don't see any reason to change terminology. Probably I'm wrong.

Reply:
Tuple is the list of objects, I think this is fine as it is.